rust/src/datatypes.rs

```diff
@@ -76,6 +76,7 @@ impl TryFrom<&DataType> for LogicalType {
             DataType::Float16 => "halffloat".to_string(),
             DataType::Float32 => "float".to_string(),
             DataType::Float64 => "double".to_string(),
+            DataType::Decimal128(precision, scale) => format!("decimal128:{precision}:{scale}"),
```

Comment:
Arrow has a Decimal256 as well, might wanna to add it?

Reply:
added